src/hooks/Tokens.ts

```diff
@@ -57,9 +59,53 @@ export function useAllTokens(): { [address: string]: Token } {
   return useTokensFromMap(allTokens, true)
 }
 
+interface BridgeInfo {
```

Comment:
I feel like this interface should be available through the token list package.

Reply:
@moodysalem thoughts on this? It crossed my mind, but I'm not sure how prescriptive you intend extensions/BridgeInfo to be.

Reply:
it's currently only an extension